### ISSUE TITLE
Missing summons for Forgotten Circles #99

### ISSUE THREAD

**Issue by cafelizardo** [PurpleKingdomGames/virtual-gloomhaven-board#77]
**Affected Scenario(s):** [e.g. 6, 10 and 33]

Scenario #99 Aftershocks from Forgotten Circles

**Describe the bug**

Bug or feature request.
1) Need to summon civilians but can only summon purple token which can be confusing if other players are summoning.  Would be nice if we could summon other colors at least.

2) When a civilian is killed *spoiler* instructions say to read section 08 which is to replace the civilian with a Valrath Savage.  Do not see a way to summon this monster.

**To Reproduce**
1. Load scenario #99.
2. Attempt to summon something to represent civilians
3. Attempt to summon Valrath Savage

**Browser :** [e.g. Chrome, Firefox]

Tried using Safari, Firefox and Chrome while using virtual Gloomhaven 1.12.0

**Comment by hobnob** [comment]
I think the first point is probably covered under #42 

Definitely the second point is a bug, which I'll get a fix for

**Comment by cafelizardo** [comment]
Replayed this scenario last night and see that we can now create the Valrath savages but still need to be able to create civilians.  I think a simple solution might be to add another monster type for this scenario - could be anything  so long as it is not already used. We ended up creating different traps so we can keep each one separate for tracking hit points.  Is there an icon available for escorts or something?

**Comment by hobnob** [comment]
Yes, I think you're right - I'll have a cast around and see if I can find any art released under Creative Commons that would be suitable for civilians. It would be nice to have tokens that actually show them
